src/DIRAC/Core/Tornado/scripts/tornado_start_all.py

```diff
@@ -14,6 +14,7 @@
 
 import os
 import sys
+
```

Comment:
Please remove all the changes to that file, since it was addressed here https://github.com/DIRACGrid/DIRAC/issues/5223